src/test/java/org/myrobotlab/service/PidTest.java

```diff
@@ -4,17 +4,22 @@
 import static org.junit.Assert.assertTrue;
 
 import org.junit.Before;
-import org.junit.Test;
+import org.myrobotlab.framework.Service;
 
-public class PidTest {
+public class PidTest extends AbstractServiceTest {
 
   @Before
   public void setUp() throws Exception {
     // LoggingFactory.init("WARN");
   }
 
-  @Test
-  public void testCompute() {
+  @Override
+  public Service createService() {
+    return (Service) Runtime.start("pid", "Pid");
+  }
+
+  @Override
+  public void testService() throws Exception {
     Pid pid = (Pid) Runtime.start("pid", "Pid");
 
```

Comment:
this.service has the pid that was created in the createService method.

Reply:
not sure what benefit there is changing the regular junit cycle into a different one.